src/main/kotlin/de/danielmaile/mpp/MPP.kt

```diff
@@ -101,6 +111,29 @@ class MPP : JavaPlugin() {
         BlockBreakingService.init()
     }
 
+    /**
+     * Send message on first install, might be changed to a better one?
+     */
+    private fun installMessage(message: String?) {
+        sendMessage(ChatColor.GREEN.toString() + "------------------------------")
+        sendMessage("")
+        sendMessage("")
+        for (s in getLanguageManager().getStringList("messages.errors.first_time_server_shutdown")) {
+            sendMessage(ChatColor.GREEN.toString() + s)
+        }
+        if(!message.isNullOrBlank()) {
+            sendMessage(message)
+        } else {
+            sendMessage("")
+        }
+        sendMessage("")
+        sendMessage(ChatColor.GREEN.toString() + "------------------------------")
+    }
+    private fun sendMessage(text: String) {
```

Comment:
Why are you using Color.GREEN inside the function and passing ChatColor.GREEN.toString() + s. Setting the color one time should be enough

Reply:
Totally true, got distracted hahah